packages/api/src/controllers/stream.test.ts

```diff
@@ -1134,15 +1170,63 @@ describe("controllers/stream", () => {
         const res = await client.get(`/stream/${document.id}`);
         expect(res.status).toBe(200);
       }
+
+      // create a new project
+      client.jwtAuth = nonAdminToken;
+      projectId = await createProject(client);
+      expect(projectId).toBeDefined();
+
+      // then create a new api-key under that project
+      const allowedOrigins = ["http://localhost:3000"];
+      newApiKey = await createApiToken({
+        client: client,
+        projectId: projectId,
+        jwtAuthToken: nonAdminToken,
+      });
+      expect(newApiKey).toMatchObject({
+        id: expect.any(String),
+      });
+
       client.jwtAuth = "";
+      client.apiKey = newApiKey.id;
+
+      // create streams with a projectId
+      for (let i = 0; i < 5; i += 1) {
+        const document = {
+          id: uuid(),
+          kind: "stream",
+          userId: nonAdminUser.id,
+          projectId: projectId,
+        };
+        await server.store.create(document);
+        const res = await client.get(`/stream/${document.id}`);
+        expect(res.status).toBe(200);
+      }
     });
 
     it("should get own streams", async () => {
       client.apiKey = nonAdminApiKey;
       let res = await client.get(`/stream/user/${nonAdminUser.id}`);
       expect(res.status).toBe(200);
       const streams = await res.json();
-      expect(streams.length).toEqual(3);
+      expect(streams.length).toEqual(8);
+      expect(streams[0].userId).toEqual(nonAdminUser.id);
+
+      // ensure project associated with a new api-key is not enforced for this endpoint
```

Comment:
Not sure I understand this comment/what is being tested here. But if we're using an API key from a ifferent project, shouldn't this API return no results? In the end game I don't think we should have any APIs that skip the project abstraction unless they're admin.